bench.sh

```diff
@@ -407,7 +411,10 @@ benchfill="           "
 function run_test_env_cmd { # <test name> <allocator name> <environment args> <command>
   echo
   echo "run $1 $2: $3 $4"
-  # cat $benchres
+  # clear temporary output
+  if [ -f $benchres.line ]; then
+    rm $benchres.line
```

Comment:
please add quotes about `$benchres.line` and prefix it with `./`, just to be on the safe side.

Reply:
So `$benchres` is a full path, so prepending with `./` does not work.  Can I ask what you want to be safe with respect to?

Reply:
If it's a full path, then no need for `./`, but quotes are still needed, should `$benchres.line` contain a space character.

Reply:
Fixed this. I think there are multiple places where this script will fail if there are spaces in the directory where it is stored.

Reply:
We should fix them all, before someone reports a "mimalloc-bench eat all my files :'("

Reply:
I fixed the ones for `rm`.  I'll look at fixing the rest.